lnwallet/reservation.go

```diff
@@ -521,6 +524,16 @@ func (r *ChannelReservation) FundingOutpoint() *wire.OutPoint {
 	return &r.partialState.FundingOutpoint
 }
 
+// SetOurUpfrontShutdown sets the upfront shutdown address on our contribution
+// and partial state in the reservation.
+func (r *ChannelReservation) SetOurUpfrontShutdown(shutdown lnwire.DeliveryAddress) {
+	r.Lock()
+	defer r.Unlock()
+
+	r.ourContribution.UpfrontShutdown = shutdown
```

Comment:
nit: As a reader it's really confusing to me that sometimes it's Shutdown, sometimes ShutdownScript. I guess this is because I'm not familiar with the code base yet.

Reply:
That's an inconsistency on my side, trying to keep var names brief. I'll settle on one or the other,